index.js

```diff
@@ -1505,6 +1505,29 @@
   }
   S.pipe = def('pipe', {f: [Z.Foldable]}, [f(Fn($.Any, $.Any)), a, b], pipe);
 
+  //# pipeK :: (Foldable f, Chain m) => f (Any -> m Any) -> m a -> m b
+  //.
+  //. Takes an array of functions assumed to be unary which return values
```

Comment:
Should this line reference "sequence" instead of "array" as well?

Reply:
I mentioned this in <https://github.com/sanctuary-js/sanctuary/pull/473#discussion_r159290807>. I don't like using “structure” here, but perhaps “sequence” is better than “array” (which is, admittedly, inaccurate). I don't have a strong opinion on this. I'm happy to stick with “array” or to change “array” to “sequence” in both descriptions. I leave this to you, Gabe.

Reply:
Sequences can be limited to a single element 😄

Reply:
I don't understand your previous comment, Gabe. :thinking: